Fix duplicate routes error

nibble0101 opened the pull request:
This PR fixes the duplicate routes error.
 ```js
warn Duplicate routes found! This could lead to non-deterministic routing behavior

Attempting to create page at /docs/howto/file-box, but a page already exists at this route
Attempting to create page at /docs/specs/, but a page already exists at this route
```
Essentially the first warning is due to two files in the [howto directory](https://github.com/wechaty/wechaty.js.org/tree/main/docusaurus/docs/howto) with the same name `file-box`.
![image](https://user-images.githubusercontent.com/52580190/140655854-a418852d-ff41-41c9-b155-04a8d165ca73.png)

I have also realized the file naming is not consistent with the side-bar label making it more difficult to maintain. The file is `file-box.mdx` but the side-bar label is `Send and receive files`.  However if the side-bar label is consistent with the file name it is easier to locate which `.md` or `.mdx` one should edit to fix an error. So I have changed the file-names accordingly. Let me know what others think.

There were two similar `file-box` files. One is `file-box.md` and another is `file-box.mdx`. I am not sure why? I think it is better to decide which one is the required  one and the other one needs to be deleted otherwise they become a source of confusion. 

@huan  I have failed to reproduce the last two errors.

 #1388

huan commented:
Thank you very much for the quick response!

I saw you renamed the `file-box`, but as my understanding, we will have a file/id named `file-box` in our folders correct?

Could you please help us to make sure that there are no orphan files in the folders? Because it seems that we have multiple file-box now.

And one more thing that, I think the `file-box` name is better for the file-name/id so please use this name if we can.

Thank you very much!

nibble0101 commented:
> Could you please help us to make sure that there are no orphan files in the folders? Because it seems that we have multiple file-box now.

Actually the warning was due to two files named `file-box` in the same directory. One was named `file-box.md` and the other `file-box.mdx`. I don't know why there are two files in the first place otherwise I would have deleted one. I have reverted the file name to `file-box.mdx` since you asked to retain the file names and renamed the other one to `file-box-two.md` to remove the warning .

huan reviewed (approved):
LGTM
